tapestry.js

```diff
@@ -48,18 +51,50 @@ jQuery.get(apiUrl + "/tapestries/" + tapestryWpPostId, function(result){
 
     tapestrySlug = dataset.settings.tapestrySlug;
     
-    if (saveProgressToCookie) {
-        // Update dataset with data from cookie (if any)
-        var cookieProgress = Cookies.get("progress-data-"+tapestrySlug);
-        if (cookieProgress !== undefined) {
-            cookieProgress = JSON.parse( cookieProgress );
-            setDatasetProgress(cookieProgress);
-        }
-        // Update H5P Video Settings from cookie (if any)
-        var cookieH5PVideoSettings = Cookies.get("h5p-video-settings");
-        if (cookieH5PVideoSettings !== undefined) {
-            cookieH5PVideoSettings = JSON.parse( cookieH5PVideoSettings );
-            h5pVideoSettings = cookieH5PVideoSettings;
+    if (saveProgress) {
+        // If user is logged in, get progress from database database
+        if (tapestryWpUserId) {
+            var getProgData = {
+                "post_id": tapestryWpPostId,
+            };
+
+            jQuery.get(TAPESTRY_PROGRESS_URL, getProgData, function(result) {
```

Comment:
Again, please do not use variables if you are only using in one place. This could just be `jQuery.get(TAPESTRY_PROGRESS_URL, { "post_id": tapestryWpPostId }, function(result) {` which is easier to read and follow. The only exception is if the variable was an object or array with many properties/members, for example in a post request. Please make sure you follow this rule in the future to avoid back & forths.

Reply:
changed to use             jQuery.get(TAPESTRY_PROGRESS_URL, { "post_id": tapestryWpPostId }, function(result) {